docs/cli-arguments.md

```diff
@@ -32,7 +32,7 @@ Usage of ./kube-state-metrics:
   -h, --help                             Print Help text
       --host string                      Host to expose metrics on. (default "0.0.0.0")
       --kubeconfig string                Absolute path to the kubeconfig file
-      --labels-allow-list string         list of metric names and labels you would like to allow, metric_name=[label1,labeln...],metric_name[]...
+      --labels-allow-list string         list of metric names and labels you would like to allow, metric_name=[label1,^label2.*,labeln...],metric_name[]...
```

Comment:
will this regex (^label2.*) get past labelsAllowListFormat.MatchString() in pkg/options/types.go?

should we update errLabelsAllowListFormat in pkg/options/types.go to be consistent?

Reply:
This regex won't pass the command-line flag parsing based on the current approach. It will take some work to get it to work, we should definitely think about this more and not approach it lightly, cc @lilic